CONTRIBUTING.md

```diff
@@ -10,35 +10,54 @@ Contributing
 Testing
 =======
 
-Running the tests requires [Grunt CLI](https://github.com/gruntjs/grunt-cli). It
-is available via [npm](https://npmjs.org):
+Install the dependencies for the project:
 
 ```
-npm install grunt-cli
+npm install
 ```
 
-Also install the dependencies for the project:
+### Headless (Using PhantomJS)
 
 ```
-npm install
+npm run test:quick
 ```
 
 ### In browser
 
+Any large changes should be tested in old IEs (we support IE 6!), and any other
+browsers you can get your hands on.
+
+#### Sauce Labs automated browser testing
+
+To do automated browser testing with sauce labs first install sauce-connect
+
 ```
-grunt browsertest
+brew cask install sauce-connect
 ```
 
-Any large changes should be tested in old IEs (we support IE 6!), and any other
-browsers you can get your hands on. The easiest way
-to get these is from [modern.ie](https://www.modern.ie/en-gb/virtualization-tools#downloads).
+Then ensure that you have `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY` environment
+variables set.
 
-### Headless (Using PhantomJS)
+Finally run:
+
+```
+karma start --single-run
```

Comment:
> I could be wrong but I think it is one of those packages that prefers to be installed globally

`karma-cli` does specify `preferGlobal`, however adding it to the `devDependencies` probably wouldn't do any harm. If anything it would make updating it in the future much simpler. And because locally installed dependencies are added to the PATH when running a script, future contributors won't have to globally install multiple versions of `karma-cli`.

Meaning `karma start --single-run` can turn into `npm test` and `npm test --browsers sl_ie_6, sl_ie_7` (a few lines down).